### PR TITLE
Add openGL X11 support to Geant4

### PR DESCRIPTION
**Pull request by Moelf**

### CODE REVIEW

File: G/Geant4/build_tarballs.jl [hunk 1]
```diff
@@ -61,6 +61,8 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     "Expat_jll",
+    "Xorg_libXmu_jll",
+    "Libglvnd_jll"
```
Comment (giordano) [inline G/Geant4/build_tarballs.jl:65]:
```suggestion
    Dependency("Expat_jll"),
    Dependency("Xorg_libXmu_jll"),
    Dependency("Libglvnd_jll"),
    BuildDependency("Xorg_xorgproto_jll"),
```

File: G/Geant4/build_tarballs.jl [hunk 2]
```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder
 
 name = "Geant4"
-version = v"0.1.0"
+version = v"0.1.1"
```
Comment (giordano) [inline G/Geant4/build_tarballs.jl:6]: Why?

Comment (Moelf) [reply G/Geant4/build_tarballs.jl:6]: don't we need to advance version of Geant4_jll so it can be differentiated?

Comment (giordano) [reply G/Geant4/build_tarballs.jl:6]: No: https://juliapackaging.github.io/BinaryBuilder.jl/latest/building/#Version-number.  But I just realised we didn't do it right the first time....  Ok, let's keep it this way for the time being

**Comment by Moelf** [comment]
@giordano thanks so much!

**Pull request merged**